validator/sawtooth_validator/execution/scheduler_parallel.py

```diff
@@ -70,24 +70,37 @@ def _tokenize_address(self, address):
             for i in range(0, len(address), self._token_size)
         ]
 
-    def _get(self, address, create=False):
-        tokens = self._tokenize_address(address)
+    def update(self, address, updater):
+        '''
+        Sets the data at ADDRESS to UPDATER(address).
```

Comment:
Should this say something like `UPDATER(data_at_address)`?

Reply:
yes